app/controllers/ops_controller/settings/zones.rb

```diff
@@ -5,62 +5,8 @@ def zone_edit
     assert_privileges(params[:id] ? "zone_edit" : "zone_new")
 
     case params[:button]
-    when "cancel"
```

Comment:
I think there are couple of other functions that can be deleted in this file, let me know if you need any help?